### DISCUSSION TITLE
New links for addins

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#3729] [Questions & Answers] [answered]
github links for meta, expenses etc do not work since changing main url. New?

**Comment by kevinpapst** [comment]
You received an invitation to a new repository, which you accepted 😄 
https://github.com/kevinpapst/kimai-plugins

So support / feature requests for all paid plugins will happen in this repository in the future.
The old links do not work any longer.
Your posts were moved to the new repository, so your ideas/questions are still existing.

**Comment by HeinzWuert** [comment, marked as answer]
sorry, found it in Spam, don't know why.
